ui/src/app/views/resource-card.tsx

```diff
@@ -484,7 +484,7 @@ export class ResourceCard extends React.Component<Props, State> {
           queryParams.set('jupyterLabMode', 'true');
         }
 
-        return `${workspacePrefix}/notebooks/${encodeURIComponent(notebook.name)}?` +
+        return `${workspacePrefix}/notebooks/preview/${encodeURIComponent(notebook.name)}?` +
```

Comment:
Is it still appropriate to include the queryParams here?  Preview/Readonly ignores them.

Actually - do we have any way to access Playground Mode or JupyterLab now?  Maybe we should remove all references to these.

Reply:
good catch. The new component doesnt do anything with it so we should just drop it.

The old notebooks page still uses them though so references to that page should append the query parameters.